day_01/day_01.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
```

Comment:
Magic shebang to allow `./day_01/day_01/py` as long as it has the executable mark on unix. Not sure on windows.